src/MovieContainer/Movie/Movie.js

```diff
@@ -4,11 +4,10 @@ import './Movie.css';
 const Movie = ({title, poster, avgRating, id, singleMovie}) => {
 
   return (
-    <article onClick={() => singleMovie(id)} className='movie'>
-      <img src={poster} width={400} height={400} />
+    <article onClick={() => singleMovie(id)}  className='movie'>
```

Comment:
Love how you're passing in the id here! 

For your image, you still want to be using the css files to handle styles changes for now. You also may not need alt text at all since you are listing the title out right below. Just make sure to have `alt=""` so the screenreader knows to skip it.